### PR TITLE
Follow feature

### PR DESCRIPTION
**Pull request by artbrock**
WIP: Enabling user follows... but no good UI working for it yet.

### CODE REVIEW

**Comment by jost-s** [comment]
@artbrock @bierlingm Follow feature works now. There are still lots of things to tweak and improve, but following and reading the mews feed works.

**Comment by jost-s** [comment]
@artbrock @bierlingm Following is done now and ready for your review. Among other things I've updated the follow icons/buttons and reverted to the default blue as primary color. A bigger change is the custom search field which for the moment only searches for usernames. It can be extended to search for everything else as well and it can be displayed any way we want. I didn't include more on this branch, it's big enough as it is.

Pending items to be done from my list:
- search for hashtags and cashtags
- search for Display Name (depends on profile component)
- show Display Name in mews feed items

The profile popup jumps after loading its content, that's a bug in the UI framework Quasar.

Lastly not there's another branch with the search function on a separate page "Discover" (Travis' work).

**Comment by jost-s** [comment]
On another topic, I've been getting console errors in the browser lately when starting `npm run network x`. Something with "GhostActor". The whole app doesn't work then, because the profile zome crashes.

I can't reproduce it yet. `npm run start` always works. Let me know if the network error happens for you as well.

**Comment by jost-s** [comment]
@bierlingm, a gentle reminder for you to skim through this PR please.

**Review by bierlingm — APPROVED** [review]
Looks good to me on skim

File: ui/src/components/MewsItemContent.vue [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="column">
+    <div class="text-body1">
+      {{ mewContent.mew.mewType.original.mew }}
```
[the remark this answers is not in the document]
Comment (jost-s) [reply ui/src/components/MewsItemContent.vue:4]: Deserves a decent refactoring, doesn't it?!

**Pull request merged**